source/partials/release_notes/_release-20-9-0.md.erb

```diff
@@ -0,0 +1,68 @@
+<h4>Stage Overview on VSM Page</h4>
+
+Staring this release, the users can utilize the stage overview on VSM page. Earlier, it was available on the Dashboard and Pipeline Activity Page.
+
+<figure class='shrink-figure-60'>
+    <%= image_tag 'releases/20.9.0/stage_overview.png', alt: 'Stage overview on VSM page' %>
+</figure>
+
+Also, the users can now view the older stage runs from the stage overview as well.
```

Comment:
In case of stage re-runs, the stage overview now provides an option to view previous stage runs.